src/components/Person/Person.jsx

```diff
@@ -1 +1,32 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+function getMarriageStatus(person) {
+  return person.sex === 'm'
+    ? `${person.partnerName} is my wife`
+    : `${person.partnerName} is my husband`;
+}
+
+export const Person = ({ person }) => (
+  <>
```

Comment:
1. it'd be nice to use object destructing 
2. fragment wrapper is redundant
```suggestion
export const Person = ({ person }) => {
  const { name, ... } = person;
  
  return (
```

Reply:
Good point! Thanks